docs/modules/ROOT/pages/index.adoc

```diff
@@ -18,6 +18,14 @@ templates. Commodore sets up an https://kapitan.dev/inventory/[inventory] by
 downloading all required components and symlinking the relevant classes into
 the inventory structure which Kapitan understands.
 
+Components can define dependencies to Jsonnet libraries using https://github.com/jsonnet-bundler/jsonnet-bundler[jsonnet-bundler].
```

Comment:
This feels a bit scarce. Shouldn't we add a bit more documentation (example, how-to, ...) about this feature, as it feels to be an important feature for the project.